examples/Misc/Program.cs

```diff
@@ -57,7 +57,7 @@ static void PrintMetadata(string brokerList)
             var config = new Dictionary<string, object> { { "bootstrap.servers", brokerList } };
             using (var producer = new Producer(config))
             {
-                var meta = producer.GetMetadata();
+                var meta = producer.GetMetadata(true, null);
```

Comment:
I agree, but what exactly is the common usage? all topics? single topic? locally used topics?
What is the common timeout?